benchmarks/pipe.py

```diff
@@ -400,7 +393,7 @@ def get_golden_config(model_name, args):
     """Return a dict with the golden data for throughput and memory usage."""
 
     if model_name == "lm":
-        return lm_wikitext2.get_golden_real_stats(args.multiprocess)
+        return lm_wikitext2.get_golden_real_stats()
```

Comment:
can we modify `benchmarks/golden_configs/lm_wikitext2.py` to remove this option?